resources/views/app/includes/sections/footer.blade.php

```diff
@@ -0,0 +1,49 @@
+<footer class="page-footer">
+  <div class="row">
+    <div class="container">
+      <div class="col 14 s7">
+        <h4 class="white-text">Testimonials</h4>
+          <ul class="collection">
+            <li class="collection-item avatar">
+              <img src="https://avatars.slack-edge.com/2015-08-03/8571445399_ce66ec6325503ab5a8f0_192.jpg" alt="" class="circle">
+              <span class="title"><b>Ibraheem</b></span>
+              <blockquote>
+                This is simply the best!.
+              </blockquote>
+        <a href="#!" class="secondary-content">
+          <i class="material-icons">grade</i>
+          <i class="material-icons">grade</i>
+          <i class="material-icons">grade</i>
+          <i class="material-icons">grade</i>
+          <i class="material-icons">grade</i></a>
```

Comment:
The closing tag `</a>` should be on a separate line. The indentation too here is off, all I can see is an anchor tag just coming out of nowhere, shouldn't it be on the same level as the `<blockquote`